test/integration/admin_can_see_orders_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+class AdminCanSeeOrdersTest < ActionDispatch::IntegrationTest
+  include CategoryItemsSetup
+  test 'logged in admin sees dashboard page with orders' do
+    admin = User.create(username: 'admin',
+                        password: 'password',
+                        role:      1)
+    create_items_associated_with_orders
+    ApplicationController.any_instance.stubs(:current_user).returns(admin)
+    visit login_path
+    fill_in 'Username', with: 'admin'
+    fill_in 'Password', with: 'password'
+    click_button 'Login'
+
+    assert page.has_content?('Welcome, Admin!')
+    within('.all-orders') do
+      assert page.has_content?('Status')
+      assert page.has_content?('Total Price')
+      assert page.has_content?('$160')
+      assert page.has_link?('View Order')
+    end
+  end
+  test 'admin sees order numbers by status' do
+    admin = User.create(username: 'admin',
+                        password: 'password',
+                        role:      1)
+    create_items_associated_with_orders
+    ApplicationController.any_instance.stubs(:current_user).returns(admin)
+    visit login_path
+    fill_in 'Username', with: 'admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.